app/controllers/results_controller.rb

```diff
@@ -111,7 +112,7 @@ def edit
       end
     end
 
-    group_order = reviewer_access ? 'grouping.id' : 'group_name'
+    group_order = reviewer_access ? 'groups.id' : 'group_name'
```

Comment:
It looks like both `group_order` and `all_groupings` are only used in the "else" case below. If so, this conditional is not necessary (just use `group_name`) and move both assignments into the `else`.